src/coreclr/src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -326,6 +326,16 @@ public static string GetTypeInfoName(ITypeInfo typeInfo)
         /// </summary>
         public static IntPtr /* IUnknown* */ GetIUnknownForObject(object o)
```

Comment:
>GetIUnknownForObject [](start = 45, length = 20)

Please file an issue to update the documentation for these functions for .NET 5. We need to make sure users understand the new interaction model.

Reply:
dotnet/dotnet-api-docs#3985